### PR TITLE
Eb 24 rds retrofit

### PR DESCRIPTION
**Pull request by ayan-usgs**

### CODE REVIEW

File: nldi-liquibase/src/main/resources/liquibase/roles.sql [hunk 1]
```diff
@@ -7,7 +7,7 @@ create role nhdplus;
 --rollback drop role if exists nhdplus;
 
 --changeset drsteini:grant.nhdplus.to.nldi
-grant nhdplus to nldi;
+grant nhdplus to postgres;
```
Comment (dsteinich) [inline nldi-liquibase/src/main/resources/liquibase/roles.sql]: The three related rollback statements should be updated too.

**Comment by ayan-usgs** [comment]
Changes to rollback made.

I've verified that the the docker-compose still works with the changes. Database seemed correctly populated when I connected to it via pgAdmin.

File: nldi-liquibase/src/main/resources/liquibase/roles.sql [hunk 2]
```diff
@@ -27,5 +27,5 @@ create role nldi_user with login password '${NLDI_USER_PASSWORD}';
 --rollback drop role if exists nldi_user;
 
 --changeset drsteini:grant.nldi_user.to.nldi_data
-grant nldi_user to nldi_data;
+grant nldi_user to postgres;
```
Comment (dsteinich) [inline nldi-liquibase/src/main/resources/liquibase/roles.sql]: This one should stay with nldi_data.

Comment (ayan-usgs) [reply nldi-liquibase/src/main/resources/liquibase/roles.sql]: Fixed.

**Comment by ayan-usgs** [comment]
All changes made. I'm going to go ahead and merge this.

**Pull request merged**